### DISCUSSION TITLE
Which release is the most stable one to get started?

### DISCUSSION THREAD

**Discussion by kevinamiri** [ianstormtaylor/slate#4597] [General]
Which release is the most stable one to get started? 
 
Just having a bit more advanced contenteditable that would work on all platforms. For example In IOS I have had 'disappearing cursor' bug. I remember in a some release I haven't had the issue, but now is came back.

I appreciate your suggestions

**Comment by dylans** [comment]
We have a few key core issues we're working through. The main current source of potential instability is the attempt to switch to native events, #3888 (similar to what React 18 did). It's introduced some issues, and we're deciding soon if we need to revert it or if we can make it work. This was introduced in 0.66.0. So perhaps the last 0.65.x release, and then wait a week or two until we have the consequences of #3888 resolved?

**Reply by dmitrizzle** [reply]
Will solving #3888 bring Slate closer to 1.0.0? I'm running a really old version that's feature rich and wondering if I can wait out the rewrite until "stable."
